tests/test_parity.py

```diff
@@ -35,27 +36,11 @@ def test_gpt_multifield_parity(self):
             atol=1e-1
         )
 
+    @pytest.mark.xfail
     def test_gpt2_featurize(self):
```

Comment:
If you know why this test is failing, could you add a comment explaining why? Or is it a failure we don't know the cause of?